app/components/inventory/rank-select.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+import SelectValues from 'hospitalrun/utils/select-values';
+
+const c = Ember.computed;
```

Comment:
I do not like use prototype extensions (`.property()`) anymore since they are globally defined and may be deprecated in the future. I like to define dependencies using `const` reference on top of file. Let me know if you're OK with this or it's against your coding style.